core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -1811,6 +1811,11 @@ public View getView(String name) {
         return viewGroupMixIn.getViews();
     }
 
+    @Exported     //Just to be exported - @Exported on default methods in interface is not supported
```

Comment:
👎  for that in the current state. The method should conduct `View.READ` permission checks at least.